dbcon/joblist/lbidlist.cpp

```diff
@@ -296,13 +297,29 @@ bool LBIDList::GetMinMax(int64_t* min, int64_t* max, int64_t* seq,
 
         if (isUnsigned(colDataType))
```

Comment:
It is safe but sUnsigned from CSC namespace returns false for DECIMAL.

Reply:
*isUnsigned

Reply:
Keeping this here for now, in case we add UDECIMAL to isUnsigned in the future.